B827EBFFFEDDF5E8.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "B827EBFFFEDDF5E8", 
+		"servers": [ 
+			{ 
+				"server_address": "router.asia-se.thethings.network", 
```

Comment:
This is still the wrong URL. Please check here for the list of routers: https://www.thethingsnetwork.org/wiki/Backend/Connect/Gateway

Reply:
I have updated the file. Please kindly check it. Thank you.